Grogs/Rhine Grogs.md

```diff
@@ -0,0 +1,311 @@
+# Rhine Grogs
+
```

Comment:
It might be worth having a general intro of who are those grogs.

Reply:
Mmm... Okay, I'll write a short intro referring to the Atlas post.